scripts/lib/zulip_tools.py

```diff
@@ -364,7 +366,8 @@ def parse_lsb_release():
         codename = subprocess_text_output(["lsb_release", "-cs"])
         distro_info = dict(
             DISTRIB_CODENAME=codename,
-            DISTRIB_ID=vendor
+            DISTRIB_ID=vendor,
+            DISTRIB_FAMILY='debian',
```

Comment:
Can you move this commit early in the branch, and then restructure `scripts/lib/create-production-venv ` and friends to use this?

Reply:
The `DISTRIB_FAMILY` commit does happen before the `create-*` scripts. This is just github commit ordering showing it otherwise.